react/components/EditorContainer/Sidebar/utils.test.ts

```diff
@@ -185,3 +185,82 @@ describe('getComponents', () => {
     })
   })
 })
+
+describe('getIsSitewide', () => {
+  const mockExtensions = {
+    'store.home': {
+      after: ['$after_footer'],
+      around: [
+        '$around_homeWrapper',
+        '$around_storeWrapper',
+        '$around_challenge',
+      ],
+      before: ['$before_header.full'],
+      blockId: 'vtex.store-theme@2.x:store.home',
+      blocks: [
+        {
+          blockId: 'vtex.store-theme@2.x:carousel#home',
+          extensionPointId: 'carousel#home',
+        },
+      ],
+      component: 'vtex.render-runtime@8.17.2/LayoutContainer',
+      composition: 'children',
+      content: {},
+      context: {},
+      implementationIndex: 0,
+      implements: [''],
+      preview: { type: 'block' },
+      props: {},
+      render: 'server',
+      track: [],
+    },
+    'store.home/$after_footer': {
+      after: [],
+      around: [],
+      before: [],
+      blockId: 'vtex.store-theme@2.x:footer',
+      blocks: [],
+      component: 'vtex.store-footer@2.6.15/index',
+      content: {},
+      contentMapId: 'gZQaBBQyU2DLvGaM9icNdg',
+      context: {},
+      implementationIndex: 0,
+      implements: [''],
+      preview: null,
+      props: {
+        showPaymentFormsInColor: false,
+        showSocialNetworksInColor: true,
+        showVtexLogoInColor: false,
+      },
+      render: 'server',
+      track: [],
+    },
+    'store.home/carousel#home': {
+      after: [],
+      around: [],
+      before: [],
+      blockId: 'vtex.store-theme@2.x:carousel#home',
+      blocks: [],
+      component: 'vtex.carousel@2.8.0/Carousel',
+      content: {},
+      contentMapId: 'eiYc7wanqAEYiPY5DJdRPT',
+      context: {},
+      implementationIndex: 0,
+      implements: [''],
+      preview: null,
+      props: {},
+      render: 'server',
+      track: [],
+    },
+  }
+
+  it('should return true for AFTER', () => {
+    const mockEditTreePath = 'store.home/$after_footer'
+    expect(getIsSitewide(mockExtensions, mockEditTreePath)).toBe(true)
+  })
+
+  it('should return false for blocks without role', () => {
```

Comment:
Missing tests for blocks that have role with AFTER, ROUND and BEFORE?

Reply:
Actually just AROUND and BEFORE. The test for AFTER is the one above